.drone.yml

```diff
@@ -40,13 +40,6 @@ steps:
     depends_on:
       - setup localstack
 
-  - name: sonar scanner
```

Comment:
This stopped working on this commit, I don't know if it's a Java 17 issue? 
I'm not holding the project back because another project can't maintain their tooling. Can we look for a github action alternative?